ksql-cli/src/main/java/io/confluent/ksql/cli/console/cmd/Exit.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2018 Confluent Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.confluent.ksql.cli.console.cmd;
+
+import io.confluent.ksql.cli.console.Console;
+import java.util.Objects;
+import org.jline.reader.EndOfFileException;
+
+class Exit implements CliSpecificCommand {
+
+  private final Console console;
+
+  Exit(final Console console) {
+    this.console = Objects.requireNonNull(console, "console");
+  }
+
+  @Override
+  public String getName() {
+    return "exit";
+  }
+
+  @Override
+  public void printHelp() {
+    console.writer().println("exit:");
+    console.writer().println(
+        "\tExit the CLI."
+    );
+  }
+
+  @Override
+  public void execute(final String commandStrippedLine) {
+    throw new EndOfFileException();
```

Comment:
This seems odd. Using an exception for non exceptional behaviour

Reply:
Not my code - just a refactor.